p3387.tex

```diff
@@ -386,7 +387,7 @@ \section{Proposed wording}
 Add a new paragraph after [expr.call], paragraph 8:
 
 \begin{adjustwidth}{0.5cm}{0.5cm}
-\added{When control is transferred out of the called function, all postcondition assertions of the function call are evaluated in sequence ([dcl.contract.func]).
+\added{When control is transferred back to this caller by the called function ([stmt.return], [expr.await]), all postcondition assertions of the function call are evaluated in sequence ([dcl.contract.func]).
```

Comment:
I don't think this is correct. An exception also transfers control back to the caller, doesn't it? We specifically want to exclude exceptions, longjmp, etc, and consider only functions that return normally, but at the same time we don't want to require a return statement, since the replacement body as currently defined doesn't have one. I don't think your wording change addresses that. (Or maybe I'm not getting it.)

Reply:
An exception transfers to an exception handler, not to the caller... but i can see that one might think "caller" is more than just the call site.  I changed to be "back to this function call" which i think is good enough for SG21 and core can wordsmith more later if they want.